scss/_global_functions.scss

```diff
@@ -58,12 +58,24 @@
   }
 }
 
-// Adds visiual focus to elements on :focus
+// Adds visual focus to elements on :focus-visible,
+// or :focus if the browser doesn't support the former
 @mixin vf-focus($color: $color-focus, $width: $bar-thickness, $has-validation: false) {
   &:focus {
     outline: $width solid $color;
     outline-offset: -#{$width};
   }
+
+  &:focus-visible {
```

Comment:
Could this be merged with focus above?

```
&:focus,
&:focus-visible {
  outline: ...
  outline-offset: ...
}
```

Reply:
I tried that too, but it didn't work - I _think_ because when compiled the selector is `foo:focus, foo:focus-visible: { ... }`, browsers that don't recognise `:focus-visible` ignore the entire thing. In practice, this meant Firefox didn't even apply `:focus` styling.

Reply:
Ah, makes sense. Thanks!